instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-jersey-2.0/javaagent/jaxrs-2.0-jersey-2.0-javaagent.gradle

```diff
@@ -14,10 +14,10 @@ dependencies {
   compileOnly group: 'javax.ws.rs', name: 'javax.ws.rs-api', version: '2.0'
   compileOnly group: 'org.glassfish.jersey.core', name: 'jersey-server', version: '2.0'
 
-  implementation project(':instrumentation:jaxrs:jaxrs-2.0:jaxrs-2.0-common')
+  implementation project(':instrumentation:jaxrs:jaxrs-2.0:jaxrs-2.0-common:javaagent')
 
-  testImplementation project(':instrumentation:servlet:servlet-3.0')
-  testImplementation project(':instrumentation:jaxrs:jaxrs-2.0:jaxrs-2.0-testing')
+  testImplementation project(':instrumentation:servlet:servlet-3.0:javaagent')
+  testImplementation project(':instrumentation:jaxrs:jaxrs-2.0:jaxrs-2.0-testing:javaagent')
```

Comment:
I don't think it makes sense to have `javaagent` sub-module in purely testing modules.